packages/turbo-test-utils/src/useFixtures.ts

```diff
@@ -12,37 +19,45 @@ interface SetupTextFixtures {
   };
 }
 
-export default function setupTestFixtures({
+export function setupTestFixtures({
   directory,
   test = "",
   options = {},
 }: SetupTextFixtures) {
   const fixtures: Array<string> = [];
-  const parentDirectory = path.join(directory, test ? test : "test-runs");
+  const parentDirectory = path.join(directory, test ? test : uuidv4());
```

Comment:
Fixes a bug that was causing multiple tests to sometimes collide when running in parallel. 

Multiple fixtures would be created under the same parent, and when one finished the entire directory would be removed. This fixes that.